portfolio/src/main/client/src/components/pages/index.ts

```diff
@@ -0,0 +1,31 @@
+import {CookingSection, DishInfo} from '@src/components/sections/cooking';
+import {htmlElement} from '@src/util/html';
+
+export const IndexPage = (): HTMLElement => {
+  const dishContext = require.context('@res/text/dish', false);
+  const dishInfos = dishContext.keys().map((key) => dishContext(key) as DishInfo);
+  const dishComponents = dishInfos.map((info) => CookingSection({info}));
```

Comment:
Why are you wrapping info in an object instead of passing it into CookingSection directly? `CookingSection(info)`

Reply:
Makes sense. Seems like you can also just add more parameters to CookingSection if you need more options. In the meantime, I'd keep this minimal and remove the object wrapping.